import-export-cli/cmd/mg/delete.go

```diff
@@ -0,0 +1,50 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package mg
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+)
+
+const deleteCmdLiteral = "delete"
+const deleteCmdShortDesc = "Delete an API in Microgateway"
```

Comment:
```suggestion
const deleteCmdShortDesc = "Delete an API"
```

Reply:
Shall we change like this? Refer [1] for further clarification

[1] https://github.com/wso2/product-apim-tooling/blob/master/import-export-cli/cmd/deleteAPI.go

Reply:
"Delete an API" was how I added it initially. Then updated it according to a [github suggestion](https://github.com/wso2/product-apim-tooling/pull/603#discussion_r573784502). I also think it is important to mention microgateway to avoid confusion with APIM delete command, from the very start. Shall we stick to "Delete an API in Microgateway"?

Reply:
Shall we do like this [1] what we have done for "apictl k8s delete"?
You can just mention
const deleteCmdShortDesc = "Delete resources related to Microgateway" 
or something which gives the above meaning.

[1] https://github.com/wso2/product-apim-tooling/blob/master/import-export-cli/cmd/k8s/delete.go

Reply:
The current APIM delete command has something like "Delete an API/APIProduct/Application in an environment". Which is why I added "Delete an API in Microgateway" because we don't have the other two. At moment I do not have a common word to represent APIs, envs, etc. Resources seems like it refers to API resources. Any suggestions?

Reply:
Yes, I got the point that you are trying to clarify. 
Even the k8s only have the ability to delete APIs, but resources do not mean API resources in there. I think since it says "Delete resources **related to kubernetes**" it has a clear meaning that it is not referring to API resources. 

Also, when we are further expanding the commands, like if we are going to implement deleting other things (not only APIs, API Products or Applications), the word resources would be suitable IMO. Further, for "apictl delete" we have kept the description as "Delete an API/APIProduct/Application in an environment" because there are 3 things to be deleted using that delete command. But, in future, if we are providing something like, "apictl delete endpoint" (just for an example), we need to change that description as well to something like "Delete resources ** from an environment**"

Reply:
I get what you mean. Since we do not have environments yet, shall I leave this as it is? Atleast until we come up with a better word other than "resources". We also have resources like listeners, clusters and etc but users won't be able to delete them because they didn't explicitly create them. I really do feel that the word resource gives different implications. Plus the only thing this delete can currently do is delete an API.

Reply:
I feel keeping it as "Delete an API in Microgateway" makes it more simple and clear